egs/wsj/s5/steps/libs/nnet3/train/chain_objf/acoustic_model.py

```diff
@@ -167,8 +167,8 @@ def train_new_models(dir, iter, srand, num_jobs,
         # work out the 1-based archive index.
         archive_index = (k % num_archives) + 1
         # previous : frame_shift = (k/num_archives) % frame_subsampling_factor
-        frame_shift = ((archive_index + k/num_archives)
-                       % frame_subsampling_factor)
+        frame_shift = int((archive_index + k/num_archives)
```

Comment:
thanks.  I think a better fix here would be to write k // num_archives.  Same for the other place-- change / to //.